client/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,5 @@
 export const environment = {
   production: true,
-  alpacaRedirectURL: "http://money-tree.tech/"
+  alpacaRedirectURL: "http://money-tree.tech/",
```

Comment:
Refactoring activity: here, you coing define one variable called url that represents "http://money-tree.tech/" and reuse it for the apiUrl with url + "api/". This can be done for the local and uat environment as well.

Reply:
Good idea, I just implemented this